src/applications/facility-locator/components/search-results/LocationPhoneLink.jsx

```diff
@@ -69,14 +54,8 @@ const LocationPhoneLink = ({ location, from, query }) => {
   } = location;
   return (
     <div>
-      {renderPhoneNumber('Main Number', null, phone.main, 'phone', null, from)}
-      {renderPhoneNumber(
-        'Mental Health',
-        null,
-        phone.mentalHealthClinic,
-        null,
-        from,
-      )}
+      {renderPhoneNumber('Main Number', null, phone.main, from)}
+      {renderPhoneNumber('Mental Health', null, phone.mentalHealthClinic, from)}
```

Comment:
`from` might have an issue by not being passed correctly but I will confirm

Reply:
Sorry @brangi - I don't follow. Please explain?